.github/workflows/tests.yml

```diff
@@ -51,6 +62,9 @@ jobs:
     # Run the tests
     - name: Run Ruby tests
       env:
+        EXERCISM_ENV: test
+        EXERCISM_CI: true
+        DYNAMODB_PORT: ${{ job.services.dynamodb.ports['8000'] }}
```

Comment:
@ErikSchierboom These are the changes needed to any GitHub thing to make it work properly through CI.

Reply:
Good to know!